### PR TITLE
Blog app: View

### PR DESCRIPTION
**Pull request by manzitresor**
### Implemented Feature

- [ ] Add Users View
- [ ] Add Post view

### CODE REVIEW

File: app/views/users/_userHeader.html.erb
Comment (KanzaTahreem) [inline app/views/users/_userHeader.html.erb:1]: - [ ] Well done for using partials 👏👏 However, kindly make sure to follow the [given wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_root_users.png?token=GHSAT0AAAAAAB5V373VVI5PWZ5PEIRSLGRUZIVZSMQ) for styling. This is how the current design looks like 👇 Don't forget to fix it for all the other pages

image

File: app/views/users/show.html.erb
Comment (KanzaTahreem) [inline app/views/users/show.html.erb:1]: - [ ] Great effort so far 👏👏Similarly here, kindly make sure to follow the [given wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAAB5V373VA3ZXWGVRCKEM4UHKZIV2LOA) for styling as per [this requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design%20from%20the%20sneak%20peek%20wireframes.). This is how the current design looks like 👇 

image

File: app/views/posts/index.html.erb
Comment (KanzaTahreem) [inline app/views/posts/index.html.erb:1]: - [ ] Here as well, please make sure to follow the given wireframe for styling as per [this requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design%20from%20the%20sneak%20peek%20wireframes.) ⛏✨

image

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -1 +1,27 @@
...
+        <h2>No Comments</h2>
+      <% end %>
+      <% post.recent_comments.each do |comment| %>
+        <li >
+          <span><%= User.find(comment.user_id).name %>:</span>
+          <span><%= comment.text %></span>
+        </li>
+      <% end %>
+    </ul>
+  <% end %>
```
Comment (KanzaTahreem) [inline app/views/posts/show.html.erb]: - [ ] Amazing effort so far👏✨ Please make sure that post show page displays post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png) ⛏ This is what we have right now 👇

image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @manzitresor 

Good job so far 👏! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the design
- ✅ Descriptive PR and README file
- ✅ Correct Linters

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1 +1,27 @@
...
```
Comment (jerryowusu) [inline app/views/posts/show.html.erb]: Nice job so far. However, as requested by the previous reviewer, kindly make sure that this show page displays all the comments of a given post and not just the 5 recent comments. 

### Your design 

Screenshot from 2023-09-28 22-13-39



### Given wireframe

Screenshot from 2023-09-28 22-17-52

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @manzitresor 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

noooo

 ## Highlights
Database working fine :+1: 
Creates users and posts :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @manzitresor , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Added Views ✔️
- Well Structured Code ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**